### ISSUE TITLE
Unable to download version 2.11 sources

### ISSUE THREAD

**Issue by thomasboussekey** [OmniDB/OmniDB#695]
Hello,

I try to upgrade OmniDB docker container to version 2.11, using https://hub.docker.com/r/taivokasper/omnidb/

In the docker file, OmniDB archive is downloaded with the command:
```sh
curl -Lo /tmp/OmniDB.zip https://github.com/OmniDB/OmniDB/archive/${OMNIDB_VERSION}.zip
```
But, this command doesn't work with the `OMNIDB_VERSION` set to `2.11.0`

Reading the [readme.md](https://github.com/OmniDB/OmniDB/blob/master/README.md) file of the project, this version exists.

Is the release tag missing?

Thanks in advance for your help,
Regards,
Thomas

**Comment by thomasboussekey** [comment]
Is it related to the fact that I don't see a branch named `Release-2.11.0`?

I tried to fork the project and create this branch from the master branch, but I was unable to **create a pull request** to initialize this branch.

Is it a correct root-cause?

Thanks for your help,
Regards,
Thomas

**Comment by rafaelthca** [comment]
Hi @thomasboussekey,

Apologies for the delay. It was actually missing the 2.11.0 Release here:

https://github.com/OmniDB/OmniDB/releases

I just created. The 2.11.0 branch is actually the master. That branch only changes when a new release is published, in that a branch for 2.11.0 is created and the master becomes 2.12.0.

Can you please try again?

**Comment by thomasboussekey** [comment]
Many thanks @rafaelthca :smile: 

I can now download OmniDB using URL: https://github.com/OmniDB/OmniDB/archive/2.11.0.zip
